src/style.css

```diff
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.paige {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.paige__body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.search-bar {
+  width: 100%;
+}
+
+.search-bar__input {
```

Comment:
Note that placeholder styles might be overwritten with browser default styles:

![image](https://github.com/user-attachments/assets/936c6f89-a383-4dfe-9d5c-c4428fe290bd)

Use ::placeholder pseudo-class to define styles for placeholder (you can use inherit value for some properties)
https://devtoolstips.org/tips/en/inspect-user-agent-dom/